brainzutils/musicbrainz_db/recording.py

```diff
@@ -20,29 +20,13 @@ def get_recording_by_mbid(mbid, includes=None):
     """
     if includes is None:
         includes = []
-    recording = get_many_recordings_by_mbid([mbid], includes)
-    return recording.get(mbid)
+    return fetch_multiple_recordings(
+        [mbid],
+        includes=includes,
+    ).get(mbid)
 
-def get_many_recordings_by_mbid(mbids, includes=None):
```

Comment:
@spellew Let's not remove public functions (without `_`) in already existing code. It might be a breaking change for code  if it's already used somewhere.